packages/libraries/main/test/scoring/guesses/calc.spec.ts

```diff
@@ -21,15 +23,15 @@ describe('scoring', () => {
       day: 14,
     }
     // @ts-ignore
-    expect(estimate(match, '1977')).toEqual({
+    expect(estimate(zxcvbnOptions, match, '1977')).toEqual({
       pattern: 'date',
       token: '1977',
       year: 1977,
       month: 7,
       day: 14,
       // @ts-ignore
       guesses: dateGuesses(match),
-      guessesLog10: 4.225050696138048,
+      guessesLog10: 4.234390722392192,
```

Comment:
Why is this changing?

I would expect the results to be the same, if this PR is just refactoring.

Reply:
I would think the same too and it has nothing to do with this changes here.
We got a new year so the date matcher scoring will match the date differently. I guess i really need to change the test date to always be x years behind and not change